symplyphysics/laws/hydro/buoyant_force_from_volume.py

```diff
@@ -0,0 +1,25 @@
+from sympy import Eq, solve
```

Comment:
Sorry but this law already exists: https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/laws/dynamics/buoyant_force_from_density_and_volume.py